hangupsbot/handlers.py

```diff
@@ -167,7 +167,10 @@ def handle_command(self, event):
 
         # check that a bot alias is used e.g. /bot
         if not event.text.split()[0].lower() in self.bot_command:
-            return
+            if self.bot.conversations.catalog[event.conv_id]["type"] == "ONE_TO_ONE" and self.bot.get_config_option('auto_alias_one_to_one'):
+                event.text = u" ".join((self.bot_command[0], event.text)) # Insert default alias if not already present
```

Comment:
There may be a more elegant way to implement this but adding the alias to the event text seemed the least likely to introduce other bugs since the post processed event is the same as it would have been if the bot had been addressed with alias.